### DISCUSSION TITLE
Post a picture of your build!

### DISCUSSION THREAD

**Discussion by sanni** [sanni/cartreader#403] [Look what I made]
Here you can upload a picture of the Cart Reader you build, bonus points for nice color combos. 🥇 

![IMG_1410](https://user-images.githubusercontent.com/19169494/178138269-c49aa8b1-24fe-4b96-b5a2-726f18ded29e.jpg)

**Comment by iidaco** [comment]
From Japan...
Thanks for great project!

![DSC_1434](https://user-images.githubusercontent.com/109296343/179009885-21257c78-60bc-4384-acc6-cb0fd1378f71.jpg)

**Comment by nsx0r** [comment]
Here's my build: HW5 Rev.3 + 6-slot adapter
I am a total noob, but it works!
Little advice for the 6-slot: solder the SNES/SFC slot 1st, then the pin headers on the underside, and then the other slots.

![Screenshot 2022-07-20 soldering](https://user-images.githubusercontent.com/13054953/179923647-7c9a72b7-e3a6-4a80-8715-fe1cb27f432b.png)
![Screenshot 2022-07-20 alive](https://user-images.githubusercontent.com/13054953/179923660-e7269221-5bd3-46d5-be1f-200884e9e2ad.png)
![Screenshot 2022-07-20 frame](https://user-images.githubusercontent.com/13054953/179923652-b5dd7aef-66f7-4b23-b0b4-19ff9900e244.png)
![Screenshot 2022-07-20 n64](https://user-images.githubusercontent.com/13054953/179923657-1d76bb0c-6f2d-483d-a903-1f99c3cb56ac.png)
![Screenshot 2022-07-20 sfc](https://user-images.githubusercontent.com/13054953/179923663-029c3f2e-45d8-4119-aca1-47a3cce49325.png)

**Comment by Icesythe7** [comment]
![PXL_20220724_112934105](https://user-images.githubusercontent.com/13612149/180644914-02d3d8e3-2874-4330-b14c-a352b6abb87e.jpg)

**Comment by Aerik72** [comment]
HW4. I took this picture for the support thread on getting Phantasy Star IV working, so I might as well post it over here. :D

Also, my adapter PCBs are loosely color-coded to system. (I remember that yellow is Game Boy by thinking of it as "Pikachu Yellow".)

![20220824_110904](https://user-images.githubusercontent.com/88597229/186583554-229c40cb-02f2-4807-992d-79deefc99825.jpg)
![20220527_125306](https://user-images.githubusercontent.com/88597229/186583759-1aaa67a2-9eea-406e-bfc6-b886db0483a5.jpg)

**Comment by mer2329** [comment]
finally got my v5 PCBs (already had the cart adapter) 
and have started my soldering 
its gonna be an all black build.
so far ive done 3 changes.
1) put a right angle header on the 5v USB port for the arduino
2) crimped a female dupont pin onto a wire to use for the 5v USB power.
3) moved the fuse on the arduino to one of the ICSP pins, (the one next to the hole, but not pin 1. not sure if thats pin 4 or 6)
this is probably not needed. but it makes me happy to know its still flashable without the adapter.

i will also be doing a socketed CIC on the adapter. and i will also be locating black ribbon cables for the display cables, and will lastly be painting the edges of the board (ill probably be using model paint as its cheap.)
this is probably not needed. but it makes me happy to know its still flashable without the adapter.

![20220924_040033](https://user-images.githubusercontent.com/6768243/192091967-609c1cc6-fb35-4d65-a687-f435c9db2065.jpg)
![20220924_035855](https://user-images.githubusercontent.com/6768243/192091978-5e375ce2-277d-4b9c-8fdc-207b7d02e79c.jpg)
![20220924_040024](https://user-images.githubusercontent.com/6768243/192091991-719e237e-8197-46d2-940c-e43a35c638e0.jpg)
![20220924_041212](https://user-images.githubusercontent.com/6768243/192092044-a83b8e74-6c89-4d4e-9da3-7c56a44f1144.jpg)

**Comment by hinakoshijyou** [comment]
HW3 V7.3
SFC SA1 was completely dumped. Many thanks for this wonderful project!:)

![P1010082 copy](https://user-images.githubusercontent.com/42428141/192994150-1d510af1-ea5b-4061-8d7b-ab8032c98837.jpg)

**Comment by PsyK0p4T** [comment]
Another kind of "build"... a (working) dumping tower !
![tower](https://user-images.githubusercontent.com/87064902/198355949-d04e7476-b531-4197-82a5-2dff7c2222af.jpg)

**Reply by cybik** [reply]
That's a tower of power all right.

**Comment by MHfromB** [comment]
![sanni_cartreader](https://user-images.githubusercontent.com/117568485/200164187-899acd02-c18b-47ea-8f29-07abb0dd4884.jpg)
I designed my own simpler base plate because I already had spacers for the display PCB.

Thanks, sanni, for making this possible!

Mirko

**Reply by mer2329** [reply]
thats an idea. and i have PLENTY of wood i can use to make a slate build.

**Comment by Stanton54** [comment]
![image](https://user-images.githubusercontent.com/120297611/207222383-d028b0fd-a5e2-4e3d-b89f-d5aeeb21184d.jpeg)

Thanks to you guys I was able to dump all these games during testing. Next step is to go to dump the first games I ever played. Thanks again!

**Comment by Chomemel** [comment]
It's alive and finished !!!
Please, don't laugh. I asked a friend: "print the case with the color you never use !" ^^

![IMG_20230113_174102 1](https://user-images.githubusercontent.com/107198559/212427337-a96fd2a8-a0aa-4457-9ae4-44cc6aad5315.jpg)
![IMG_20230113_172417 1](https://user-images.githubusercontent.com/107198559/212428204-38a044ac-f996-4102-854c-9d516ffafe9d.jpg)

Thanks a lot to everyone than helped me; especially Sanni for all your answers to my questions. Thanks for sharing this incredible project.

**Reply by sanni** [reply]
Zelda Edition 🥇

**Reply by 7ank0v1c** [reply]
Looks great 😊 

love the lcd colours

**Comment by josue-clavijo** [comment]
Hi guys! Here I am to report that my build of V. 5 Cart reader has been a totally successful project. I'm a some kind noob for electronics, soldering and building devices, so my build will never gonna win design/style prize/award, but I've managed to get all parts together and working. I'm very pleased with the result: A very heartwarming (at least for me) pack of NES, Famicom, SNES an N64 cartridges have been dumped, and even no single one could be found as an impossible task. All of the dumped roms are funcional and corresponding to database CRC's. 

It is worthy to mention that in some cases I'had to try the dump more than several times, but after checkin' cart slot contacts or pcb's connections, all dumping jobs went OK.

I do attach some pics of the dumped carts as proof of completion of the whole list of dumped GEMS:
![IMG20230107131615](https://user-images.githubusercontent.com/122684684/212479921-4abd241a-16bb-4be8-9b66-ad6618fb0618.jpg)

![IMG20230107132022](https://user-images.githubusercontent.com/122684684/212479931-9dfc234e-f32e-4236-b815-2846cdc40fbf.jpg)

![IMG20230107132626](https://user-images.githubusercontent.com/122684684/212479951-46b55438-b070-4bb0-a33a-d8bbaf4c3c6a.jpg)

![IMG20230107182354](https://user-images.githubusercontent.com/122684684/212479960-53735d7c-7bd9-41e2-8b47-6c170de873b2.jpg)

![IMG20230108150351](https://user-images.githubusercontent.com/122684684/212479964-e7951b3d-2fb6-40a0-8f7e-23da75bd27a1.jpg)

![IMG20230112185657](https://user-images.githubusercontent.com/122684684/212479979-b6efe424-22af-40e8-8a2a-05904d5fc462.jpg)


FULL LIST:


![IMG20230114101236](https://user-images.githubusercontent.com/122684684/212479992-0a5ae05c-5657-4704-b5b3-f07549de5f53.jpg)



![IMG20230114101339](https://user-images.githubusercontent.com/122684684/212479998-eff3a03f-b283-4cff-a255-160b58eee24f.jpg)



The dumped games are nowadays played by me on my Anbernic Win600 handheld (Retroarch on Steam AND Batocera via USB), on RetroArch for **_the unnamed_** (Using a SteelSeries Stratus XL Controller via BT dongle), and on RetroArch in a spare Android phone.

Nota Bene: The N64 cartridge is slightly off match with the pin holes, so it's not soldered but as it fits very tight into the pcb holes, the N64 cartridge reads sweetly.

THANKS TO SANNI AND ALL OTHER CODERS/CONTRIBUTORS/BUILDERS for making possible this so awesome Cart Reader Marvel!!!

Best Wishes and Have a happy new year for all of you folks, 

Jessie

**Comment by Br1ce621** [comment]
Would anyone consider selling these? I'm interested but don't have to tech knowhow to do it. Also does this essentially make the mister a game console as in set it up put in the cartridge turn on the mister and play whatever game?

**Reply by sanni** [reply]
The OSCR just dumps the ROM to the SD card. There is no real-time access to the cartridge over the USB port, therefore you can't connect it to the MiSTer.

Also please understand that this Github is for development and not a marketplace.

**Comment by Seroczynski** [comment]
![1PypU6vpGhw4ZthLhkb98NR49VNCFxqXQmxLaqgU5m9PJP21EB](https://user-images.githubusercontent.com/10141349/223392778-6c64f1d2-ed37-4345-995e-c48bf137697a.png)

I'm making five of these, all looking the same for obvious reasons. Loving them!

**Comment by lucadentella** [comment]
Just completed! Works perfectly... **thanks a lot for sharing the project!!!**

![sanni](https://github.com/sanni/cartreader/assets/1087635/aa270263-2ba2-4030-b224-5eb96d28b028)

**Comment by Ancyker** [comment]
Finally got this built today :3

![2024-06-30 13 16 22](https://github.com/sanni/cartreader/assets/7105333/0175b509-6d92-45b1-bc5a-2e8805d55de2)

![2024-06-30 13 17 08c](https://github.com/sanni/cartreader/assets/7105333/f49b18d9-4c57-48b9-bee9-f4704d5fb8fa)

**Comment by rinemetal** [comment]
Used a carbon fiber pattern print bed and some silk pla to pull off a carbon and anodized look.

![IMG_0190](https://github.com/user-attachments/assets/07b546ea-a9e4-4c3a-90ef-f7511a1fbbff)
![IMG_0187](https://github.com/user-attachments/assets/fd0262e4-2671-4b57-b433-666a8c00d9a0)
![IMG_0192](https://github.com/user-attachments/assets/c9c88c8b-7299-4066-a5c5-37036c58bf6b)

**Comment by StanMOI** [comment]
Sanni Cart Reader HW5 Wooden Style Enclosed Shell
![Sanni Cart Reader HW5 Enclosed wood Shell (@stanMOI25) 9](https://github.com/user-attachments/assets/811bbbcb-b3de-416c-944c-d06c935ed280)
![Sanni Cart Reader HW5 Enclosed wood Shell (@stanMOI25) 2](https://github.com/user-attachments/assets/964e915e-c887-49aa-a635-8d8a64effecb)
![Sanni Cart Reader HW5 Enclosed wood Shell (@stanMOI25) 5](https://github.com/user-attachments/assets/4a4cc7a8-1e05-4a83-8d89-f3dc9971d950)
![Sanni Cart Reader HW5 Enclosed wood Shell (@stanMOI25) 6](https://github.com/user-attachments/assets/962479df-be50-4d5d-90b0-4b13a5e714ec)
![Sanni Cart Reader HW5 Enclosed wood Shell (@stanMOI25) 7](https://github.com/user-attachments/assets/5f54ff68-9b81-43ab-9634-0530de6684a5)
![Sanni Cart Reader HW5 Enclosed wood Shell (@stanMOI25) 8](https://github.com/user-attachments/assets/304437be-12c9-49b1-a033-81edde0c8127)

**Reply by Chomemel** [reply]
Gorgeous !!! I like the mix electronics + wood. And what an excellent solution to close the shell. 
Bravo.

**Comment by SourceK78** [comment]
A combination of  KytorIndustries' fully enclosed shell and a selfmade adapter shell.  

![oscr_ws_adapter_shell](https://github.com/user-attachments/assets/4b1f74b0-592d-4fb1-a55a-1b28a32c6d28)  
![oscr_pce_adapter_shell](https://github.com/user-attachments/assets/7701e986-4a68-42f0-a530-e1349fe2ab41)  
![oscr_ngp_adapter_shell](https://github.com/user-attachments/assets/ec6ec4ba-a439-434c-98cf-e4fc8dc7fc72)  

The adapter shell can be downloaded from the following URL.  

**Wonderswan**  
[https://sourcek.booth.pm/items/7010825](https://sourcek.booth.pm/items/7010825)  

**PC-Engine / TurboGrafx-16**  
[https://sourcek.booth.pm/items/7100217](https://sourcek.booth.pm/items/7100217)  

**Neogeo Pocket**  
[https://sourcek.booth.pm/items/7107347](https://sourcek.booth.pm/items/7107347)

**Reply by sanni** [reply]
Looks amazing 😍

**Reply by SourceK78** [reply]
Thank you for your reply.
It seems that the model at the link requires you to create an account even just to download it, so I have also published the model on Printables.
[https://www.printables.com/@SourceK_3397429](https://www.printables.com/@SourceK_3397429)
